### ISSUE TITLE
POS Towers break client upon login

### ISSUE THREAD

**Issue by jdhirst** [EvEmu-Project/evemu_Crucible#159]
**Describe the bug**
For some reason when you login with a POS tower next to you, the client freaks out. 

This has been reproducible most of the time; and as long as you don't login next to a POS tower, you should be fine.

**Comment by jdhirst** [comment]
The client dumps the following errors while this issue occurrs:
```
9906	2021.10.16 19:04:03:866	StoreState: 1177 midTick: False
9907	2021.10.16 19:04:03:866	State entry [<BlueWrapper: blue.MemStream (4B43F1D8)>, 1177, False]
9908	2021.10.16 19:04:03:871	EXCEPTION #17 logged at  10/16/2021 21:04:03 ; caught by eve.ExceptionHandler
9909	2021.10.16 19:04:03:871	Context info: BluePyOS::PyError()
9910	2021.10.16 19:04:03:871	Caught at:
9911	2021.10.16 19:04:03:871	/client/script/remote/michelle.py(900) DoPreTick
9912	2021.10.16 19:04:03:871	Thrown at:
9913	2021.10.16 19:04:03:871	/client/script/remote/michelle.py(900) DoPreTick
9914	2021.10.16 19:04:03:871	/client/script/remote/michelle.py(1330) RealFlushState
9915	2021.10.16 19:04:03:871	/client/script/remote/michelle.py(1062) SetState
9916	2021.10.16 19:04:03:871	slimItem = <slimItem: itemID=140000940,ballID=0,charID=None,ownerID=98000000,typeID=27897,groupID=707,categoryID=23>
9917	2021.10.16 19:04:03:871	ball = <BlueWrapper: destiny.ClientBall (4D2C9008)>
9918	2021.10.16 19:04:03:871	allyTickersToPrime = [99000000, 99000003]
9919	2021.10.16 19:04:03:871	effectInfo = ('OnSpecialFX', (...))
9920	2021.10.16 19:04:03:871	k = 40053759
9921	2021.10.16 19:04:03:871	stuffToAdd = [(...),
9922	2021.10.16 19:04:03:871	(...),
9923	2021.10.16 19:04:03:871	(...),
9924	2021.10.16 19:04:03:871	(...),
9925	2021.10.16 19:04:03:871	...
9926	2021.10.16 19:04:03:871	ownersToPrime = [90000005, 98000001]
9927	2021.10.16 19:04:03:872	self = <destiny.Ballpark object at 0x49909930>
9928	2021.10.16 19:04:03:872	locationsToPrime = [40053706L,
9929	2021.10.16 19:04:03:872	40053707L,
9930	2021.10.16 19:04:03:872	40053708L,
9931	2021.10.16 19:04:03:872	40053709L,
9932	2021.10.16 19:04:03:872	...
9933	2021.10.16 19:04:03:872	bag = <Anonymous KeyVal: {'stamp': 1177, 'damageState': {40053760: ((1.0, 110000.0, 132788846390181872L), 1.0, 1.0), 40053761: ((1.0, 110000.0, 132788846390182000L), 1.0, 1.0), 40053762: ((1.0, 110000.0, 132788846390182144L), 1.0, 1.0), 4005376-
9934	2021.10.16 19:04:03:872	3: ((1.0, 110000.0, 132788846390182288L), 1.0, 1.0), 40053764: ((1.0, 110000.0, 132788846390182400L), 1.0, 1.0), 40053765: ((1.0, 110000.0, 132788846390182576L), 1.0, 1.0), 40053766: ((1.0, 110000.0, 132788846390182688L), 1.0, 1.0), 40053767: ((1.0-
9935	2021.10.16 19:04:03:872	, 110000.0, 132788846390182832L), 1.0, 1.0), 40053768: ((1.0, 110000.0, 132788846390182976L), 1.0, 1.0), 40053769: ((1.0, 110000.0, 132788846390183088L), 1.0, 1.0), 40053770: ((1.0, 110000.0, 132788846390183264L), 1.0, 1.0), 40053771: ((1.0, 110000-
9936	2021.10.16 19:04:03:872	.0, 132788846390183392L), 1.0, 1.0), 40053772: ((1.0, 110000.0, 132788846390183584L), 1.0, 1.0), 40053773: ((1.0, 110000.0, 132788846390183696L), 1.0, 1.0), 40053774: ((1.0, 110000.0, 132788846390183872L), 1.0, 1.0), 40053775: ((1.0, 110000.0, 1327-
9937	2021.10.16 19:04:03:872	88846390184128L), 1.0, 1.0), 40053776: ((1...
9938	2021.10.16 19:04:03:872	ms = <BlueWrapper: blue.MemStream (48888F60)>
9939	2021.10.16 19:04:03:872	v = ((...), 1.0, 1.0)
9940	2021.10.16 19:04:03:872	now = 132788846435078948L
9941	2021.10.16 19:04:03:872	tickersToPrime = [98000000, 98000006, 98000001]
9942	2021.10.16 19:04:03:872	TypeError: OnSpecialFX() takes at least 11 arguments (3 given)
9943	2021.10.16 19:04:03:872	Thread Locals:  session was <Session: (sid:5374880829936278562, clientID:0, mutating:0, locationid:30000848, corprole:0xffffe07ffffff81L, userid:1, languageID:EN, role:0x63f8000280c40000L, charid:90000005, address:172.16.15.253:50660, userType:30, sess-
9944	2021.10.16 19:04:03:872	ionType:5, regionid:10000010, constellationid:20000124, allianceid:99000000, warfactionid:0, corpid:98000001, shipid:140000582, solarsystemid:30000848, solarsystemid2:30000848, hqID:60014137, baseID:60014137, rolesAtAll:0xffffe07ffffff81L, rolesAtH-
9945	2021.10.16 19:04:03:872	Q:0xffffe07ffffff81L, rolesAtBase:0xffffe07ffffff81L, rolesAtOther:0xffffe07ffffff81L, genderID:1, bloodlineID:1, raceID:1, corpAccountKey:1000)>
9946	2021.10.16 19:04:03:872	EXCEPTION END
9947	2021.10.16 19:04:03:872	Python handled exception: BluePyOS::PyError()
9948	2021.10.16 19:04:03:887	GetBallpark object <destiny.Ballpark object at 0x49909930> now has: 11 references
9949	2021.10.16 19:04:03:913	EXCEPTION #18 logged at  10/16/2021 21:04:03 Exception in Error Logging, preventing infinite recursion
9950	2021.10.16 19:04:03:913	Caught at:
9951	2021.10.16 19:04:03:913	/common/lib/bluepy.py(86) CallWrapper
9952	2021.10.16 19:04:03:913	/../carbon/common/script/sys/alert.py(585) SendStackTraceAlert_thread
9953	2021.10.16 19:04:03:913	Thrown at:
9954	2021.10.16 19:04:03:913	/../carbon/common/script/sys/alert.py(583) SendStackTraceAlert_thread
9955	2021.10.16 19:04:03:914	/../carbon/common/script/sys/alert.py(641) __SendStackTraceAlert
9956	2021.10.16 19:04:03:914	locationID1 = None
9957	2021.10.16 19:04:03:914	origin = 3
9958	2021.10.16 19:04:03:914	locationID2 = None
9959	2021.10.16 19:04:03:914	stackTrace = "EXCEPTION #17 logged at  10/16/2021 21:04:03 ; caught by eve.ExceptionHandler\nContext info: BluePyOS::PyError()\nCaught at:\n/client/script/remote/michelle.py(900) DoPreTick\nThrown at:\n/client/script/remote/michelle.py(900) DoP-
9960	2021.10.16 19:04:03:914	reTick\n/client/script/remote/michelle.py(1330) RealFlushState\n/client/script/remote/michelle.py(1062) SetState\n        slimItem = <slimItem: itemID=140000940,ballID=0,charID=None,ownerID=98000000,typeID=27897,groupID=707,categoryID=23>\n        -
9961	2021.10.16 19:04:03:914	ball = <BlueWrapper: destiny.ClientBall (4D2C9008)>\n        allyTickersToPrime = [99000000, 99000003]\n        effectInfo = ('OnSpecialFX', (...))\n        k = 40053759\n        stuffToAdd = [(...),\n                        (...),\n               -
9962	2021.10.16 19:04:03:914	(...),\n                        (...),\n                       ...\n        ownersToPrime = [90000005, 98000001]\n        self = <destiny.Ballpark object at 0x49909930>\n        locationsToPrime = [40053706L,\n                        40053-
9963	2021.10.16 19:04:03:914	707L,\n                        40053708L,\n      ...
9964	2021.10.16 19:04:03:914	i = 0
9965	2021.10.16 19:04:03:914	GetArgumentsFromStackTrace = <function GetArgumentsFromStackTrace at 0x49BBDAB0>
9966	2021.10.16 19:04:03:914	self = <svc.Alert instance at 0x490FCE90>
9967	2021.10.16 19:04:03:914	userID = None
9968	2021.10.16 19:04:03:914	nodeID = None
9969	2021.10.16 19:04:03:914	nextErrorKeyHash = None
9970	2021.10.16 19:04:03:914	mode = 'Error'
9971	2021.10.16 19:04:03:914	charID = None
9972	2021.10.16 19:04:03:914	stackID = (-1648341419,
9973	2021.10.16 19:04:03:914	'Caught at:\n/client/script/remote/michelle.py(900) DoPreTick\nThrown at:\n/client/script/remote/michelle.py(900) DoPreTick\n/client/script/remote/michelle.py(1330) RealFlushState\n/client/script/remote/michelle.py(1062) SetStat-
9974	2021.10.16 19:04:03:914	e\n')
9975	2021.10.16 19:04:03:914	TypeError: 'NoneType' object is not iterable
9976	2021.10.16 19:04:03:914	Thread Locals:  session was <Session: (sid:5374880829936278562, clientID:0, mutating:0, locationid:30000848, corprole:0xffffe07ffffff81L, userid:1, languageID:EN, role:0x63f8000280c40000L, charid:90000005, address:172.16.15.253:50660, userType:30, sess-
9977	2021.10.16 19:04:03:914	ionType:5, regionid:10000010, constellationid:20000124, allianceid:99000000, warfactionid:0, corpid:98000001, shipid:140000582, solarsystemid:30000848, solarsystemid2:30000848, hqID:60014137, baseID:60014137, rolesAtAll:0xffffe07ffffff81L, rolesAtH-
9978	2021.10.16 19:04:03:914	Q:0xffffe07ffffff81L, rolesAtBase:0xffffe07ffffff81L, rolesAtOther:0xffffe07ffffff81L, genderID:1, bloodlineID:1, raceID:1, corpAccountKey:1000)>
9979	2021.10.16 19:04:03:914	EXCEPTION END
```

**Comment by jdhirst** [comment]
This is fixed as of my testing on 850d2105e01330fddc9cb52338fb5baf84641d52. Closing.
